docker-compose.yml

```diff
@@ -1,15 +1,43 @@
-app:
-  build: .
-  links:
-   - db:db
-   - redis:redis
-  ports:
-   - "8080:8080"
-db:
-  image: mongo
-  ports:
-    - "27017:27017"
-redis:
-  image: redis
-  ports:
-    - "6379:6379"
+version: '2'
+services:
+  app:
+    hostname: app
+    build:
+      context: .
+    depends_on:
+      - db
+      - redis
+    restart: always
+    networks:
+      - hublin-net
+  db:
+    hostname: db
+    image: mongo
+    restart: always
+    networks:
+      - hublin-net
+  redis:
+    hostname: redis
+    image: redis
+    restart: always
+    networks:
+      - hublin-net
+  haproxy:
```

Comment:
Sorry but we don't want to force usage of haproxy in this docker-compose file.
I suggest to open a new pull-request to provide a new `docker-compose-haproxy.yml` file which provide this feature.